Support "application/graphql" Content-Type header

sirsavary opened the pull request:
Ran into some issues with certain GraphQL servers that do not return a JSON Content-Type header.

schickling commented:
Thanks for this PR @sirsavary. Can you point me to some of these GraphQL servers?

To my knowledge `application/graphql` can be used for the `Content-Type` of incoming GraphQL requests but the outgoing `Content-Type` is always `application/json`.

kbrandwijk commented:
As @schickling pointed out, `application/graphql` is meant for requests, not responses, as responses are valid JSON, and _not_ GraphQL, so the content-type is actually wrong.

sirsavary commented:
Interesting, the server / middleware in question is https://github.com/coopnd/fastify-apollo/blob/master/graphql.js#L19 which I forked and rewrote. 

Looking at other GraphQL implementations they do in fact return `application/json` on responses. I'm not entirely sure why `fastify-apollo` returns the GraphQL content type on responses but I copied that design choice into my fork, not realizing this wasn't the standard.

I've now updated my rewrite to return the expected header and `graphql-request` is working as expected.
